ledger/src/blockstore.rs

```diff
@@ -2015,8 +2017,19 @@ impl Blockstore {
                 ("num_repaired", num_repaired, i64),
                 ("num_recovered", num_recovered, i64),
             );
+
+            // send
+            if let Some(ref sender) = self.shred_timing_point_sender {
+                info!("PohTimingPoint:Full {}", slot);
```

Comment:
I don't think we need this message enabled by default. The is_full already prints here.